CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing to æSophia
```

Comment:
I think it should be aesophia...

Reply:
Just "Sophia" would be the safest

Reply:
Done here 321edb0e7bdb70f8b3611245a519131aea9b70c9